.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
```

Comment:
let's use pre-commit hook only for changed files (npx lint-staged )
also don't forget to add this to the package.json: 
  "lint-staged": {
    "src/**/*.{js,jsx,ts,tsx}": "eslint"
  },

Reply:
but we can try you solution